Numerous sources suggest just to do this, boost devs only seem to "know"
it is "bad".  Without this, boost simply dies compiling (joy).

--- boost/math/tools/config.hpp
+++ boost/math/tools/config.hpp
@@ -43,7 +43,7 @@
 // Darwin's rather strange "double double" is rather hard to
 // support, it should be possible given enough effort though...
 //
-#  define BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS
+// fsck it! #  define BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS
 #endif
 #if defined(unix) && defined(__INTEL_COMPILER) && (__INTEL_COMPILER <= 1000)
 //
